packages/belos/tpetra/test/BlockCG/CMakeLists.txt

```diff
@@ -21,38 +21,24 @@ TRIBITS_ADD_EXECUTABLE_AND_TEST(
   COMM serial mpi
   )
 
-#TRIBITS_ADD_EXECUTABLE_AND_TEST(
-#  Tpetra_BlockCG_complex_hb_test
-#  SOURCES test_bl_cg_complex_hb.cpp
-#  ARGS
-#  COMM serial mpi
-#  )
+ASSERT_DEFINED(Teuchos_ENABLE_COMPLEX)
```

Comment:
`Tpetra_INST_COMPLEX_DOUBLE` is the CMake option we should use here.  (Tpetra forbids setting that option without having `Teuchos_ENABLE_COMPLEX` set.)